test/java/seedu/addressbook/storage/InvalidData.txt

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<AddressBook>
+    <persons>
+        <name>John Doe</name>
+
+        <email isPrivate="false">johnd@gmail.com</email>
+        <address isPrivate="false">John street, block 123, #01-01</address>
+    </persons>
+</AddressBook>
```

Comment:
Is this meant to be valid xml but invalid data, or invalid xml? Either way, make it more obvious? e.g use content that is obviously invalid. For example, this is clearly not even xml
```
This is some random text, not even xml
``` 
This is valid xml, but invalid data
```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<Invalid>
    <data>blah</data>
</Invalid>
```